src/reference/formal_and_informative/specB12.rst

```diff
@@ -29,7 +29,7 @@
   :start-after: marker_math_4
   :end-before: marker_math_end
 
-.. include:: ../libcellml/libcellmlB12_math.rst
+.. todo ../libcellml/libcellmlB12_math.rst
 
 
 
```

Comment:
Delete those empty lines. Same in other files in fact.

Reply:
0ad0d51